### ISSUE TITLE
prevent-abbreviations: when multiple replacements are defined, suggest each through ESLint's suggest API

### ISSUE THREAD

**Issue by felixfbecker** [sindresorhus/eslint-plugin-unicorn#767]
Currently, the `prevent-abbreviations` autofixer only applies when there is only a single replacement defined. But when multiple replacements are there (if the abbreviation is ambiguous) the violation has to be fixed manually. 

It would be nice if the replacements were all suggested to the user in the editor/IDE to pick from. ESLint has the `suggest` API for this: https://eslint.org/docs/developer-guide/working-with-rules#providing-suggestions

**Comment by fisker** [comment]
I thought about this before, we can also auto fix fixable words and combine rest words as suggestion.

Example: `resArr`, we suggest(I mean suggest in message) `responseArray` and `resultArray`, without fix.

We should fix it to `resArray`, and then suggest `responseArray` and `resultArray`.

I also thought about report one word at a time, but not sure is that good.

**Comment by felixfbecker** [comment]
I don't think that would be a good idea for converting large codebases, because it will be hard to catch and correct the "wrong" fixes. I would prefer the workflow where I can first run `eslint --fix` to fix all _unambiguous_ abbreviations, then go through the rest of the failures and choose from the suggestions. I think autofixing an ambiguous abbreviation to a potentially wrong option would also be disruptive when having autofix-on-save enabled in the editor.

**Comment by papb** [comment]
@felixfbecker But is it possible that `resArr` being converted to `resArray` constitutes a wrong fix?

**Comment by felixfbecker** [comment]
No I think that's unambiguous. Not sure what you're getting at?
